pkg/config/pull_policy.go

```diff
@@ -76,13 +76,13 @@ func (p PullPolicy) Validate() error {
 // * "never"   <-> PullPolicyNever
 func ParsePullPolicy(s string) (PullPolicy, error) {
 	switch s {
-	case "always":
+	case "always", "Always":
```

Comment:
Why not just do ToLower(), that way if someone eventually supports ALWAYS, you have it handled.

Reply:
I did `ToLower()` in Podman but @cdoern brought up that K8s may not accept a YAML with a typo (e.g., AlwaYs).  That may lead to a situation where Podman would happily eat a YAMl that K8s would later on reject.

Reply:
Of course if kube does accept ALWAYS or AlwaYs, then we still have a bug.